bills/views.py

```diff
@@ -1,5 +1,46 @@
+# pylint: disable=too-many-ancestors,no-member, unused-argument
 """Views for bills application."""
 
-# from django.shortcuts import render
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
 
-# Create your views here.
+from bills import models, serializers
+
+
+class ParticipantViewset(viewsets.ModelViewSet):
+    """Participant viewset"""
+
+    queryset = models.Participant.objects.all()
+    serializer_class = serializers.ParticipantSerializer
+
+
+class EventViewset(viewsets.ModelViewSet):
+    """Event viewset"""
+
+    queryset = models.Event.objects.all()
+    serializer_class = serializers.EventSerializer
+
+    @action(detail=True, methods=["GET"])
+    def event_participants(self, request, *args, **kwargs):
```

Comment:
It should not be an action. Please use drf-nested-routers library (https://github.com/alanjds/drf-nested-routers) and `Participant`, `Bill` and `Payment` views under `api/events/{id}/"` endpoint

Reply:
https://blog.apptension.com/2017/09/13/rest-api-using-django-rest-framework/

Reply:
Done